### ISSUE TITLE
Your video file was too large (max. size: 8G)" error when uploading video

### ISSUE THREAD

**Issue by l0u1sg** [Chocobozzz/PeerTube#4969]
**Describe the current behavior**

Since today, when I try to upload a video, I get this error from PeerTube
![image](https://user-images.githubusercontent.com/46707587/167248200-fc78c899-cacc-4e93-b034-a89e8a383491.png)

The error didn't appear before and the file I'm trying to upload is much smaller than 8Gb. I saw that a way out had already been opened on this, and the technique of clicking retry every time works, but it is inconvenient. 

**Steps to reproduce:**

1. Connect to Peertube
2. Upload a 500mo video

**Describe the expected behavior**
The video is much less than 8Gb and should therefore send without problem

**Additional information**

* PeerTube instance:
  * URL: https://peertube.louis.software
  * version: 4.1.1
  * NodeJS version: 14.19.1
  * The instance is running the latest version of Docker with the chocobozzz/peertube:production-bullseye image on an ARM architecture
  * The instance runs behind a reverse proxy on Nginx Proxy Manager, without additional configuration. The proxy is also running behind Cloudflare
* Browser name, version and platforms on which you could reproduce the bug: Safari/Chrome/Firefox
* Link to browser console log if relevant:
![image](https://user-images.githubusercontent.com/46707587/167248621-d61f86fe-520c-439c-a89a-58074ea6bb31.png)

* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):
```
[peertube.louis.software:443] 2022-05-07 09:34:19.563 info: 78.194.X.X - - [07/May/2022:09:34:19 +0000] "PUT /api/v1/videos/upload-resumable?upload_id=370cabd5-95512da1-5e4a1f07-242fa213 HTTP/1.1" 308 - "https://peertube.louis.software/videos/upload" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Safari/605.1.15"
[peertube.louis.software:443] 2022-05-07 09:34:19.911 info: 78.194.X.X - - [07/May/2022:09:34:19 +0000] "PUT /api/v1/videos/upload-resumable?upload_id=370cabd5-95512da1-5e4a1f07-242fa213 HTTP/1.1" 308 - "https://peertube.louis.software/videos/upload" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Safari/605.1.15"
[peertube.louis.software:443] 2022-05-07 09:34:20.140 info: 78.194.X.X - - [07/May/2022:09:34:20 +0000] "PUT /api/v1/videos/upload-resumable?upload_id=370cabd5-95512da1-5e4a1f07-242fa213 HTTP/1.1" 308 - "https://peertube.louis.software/videos/upload" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Safari/605.1.15"
[peertube.louis.software:443] 2022-05-07 09:34:20.678 info: 78.194.X.X - - [07/May/2022:09:34:20 +0000] "PUT /api/v1/videos/upload-resumable?upload_id=370cabd5-95512da1-5e4a1f07-242fa213 HTTP/1.1" 308 - "https://peertube.louis.software/videos/upload" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Safari/605.1.15"
[peertube.louis.software:443] 2022-05-07 09:34:21.773 info: 78.194.X.X - - [07/May/2022:09:34:21 +0000] "PUT /api/v1/videos/upload-resumable?upload_id=370cabd5-95512da1-5e4a1f07-242fa213 HTTP/1.1" 308 - "https://peertube.louis.software/videos/upload" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.4 Safari/605.1.15"
```

**Comment by kukhariev** [comment]
@l0u1sg  Cloudflare workaround https://github.com/Chocobozzz/PeerTube/pull/4857

**Comment by l0u1sg** [comment]
@kukhariev Ok, but where am I supposed to put it in my configuration? Because when I add it in my local-production.json, the line is removed as soon as the PeerTube container is restarted

**Comment by l0u1sg** [comment]
Damn, I didn't see that a pull request had been opened.

**Comment by l0u1sg** [comment]
Does it correspond to this in the env for Docker?
[  max_upload_part:](https://github.com/Chocobozzz/PeerTube/blob/439567eb8aeefa8a91e36ed3d7f907280f6a4dfe/support/docker/production/config/custom-environment-variables.yaml#L61)

**Comment by Chocobozzz** [comment]
Hello,

The Cloudflare workaround will be available in next PeerTube version (4.2.x) :)
